chain/x/oracle/genesis.go

```diff
@@ -12,9 +12,10 @@ import (
 
 // GenesisState is the oracle state that must be provided at genesis.
 type GenesisState struct {
-	Params        types.Params         `json:"params" yaml:"params"`
-	DataSources   []types.DataSource   `json:"data_sources"  yaml:"data_sources"`
-	OracleScripts []types.OracleScript `json:"oracle_scripts"  yaml:"oracle_scripts"`
+	Params        types.Params              `json:"params" yaml:"params"`
+	DataSources   []types.DataSource        `json:"data_sources"  yaml:"data_sources"`
+	OracleScripts []types.OracleScript      `json:"oracle_scripts"  yaml:"oracle_scripts"`
+	Reporters     map[string]sdk.ValAddress `json:"reporters" yaml:"reporters"`
```

Comment:
You are trying to use a map from the reporter to the validator, but our chain spec 1 reporter address can be a reporter for multiple validators. (In practice, it doesn't happen) but my recommendation is to create a new data type Reporters (validator_address sdk.ValAddress, reporters []sdk.AccAddress)

Reply:
Got it. Fixed.